go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/kr/pretty v0.2.2-0.20200810074440-814ac30b4b18 // indirect
 	github.com/kr/text v0.1.0 // indirect
 	go.mozilla.org/pkcs7 v0.0.0-20200128120323-432b2356ecb1 // indirect
+	golang.org/x/exp v0.0.0-20230905200255-921286631fa9
```

Comment:
we need to see if this is packaged on the distributions where we are not vendoring

Reply:
actually this is not used atm?